pom.xml

```
@@ -15,7 +15,7 @@
     <parent>
         <groupId>com.powsybl</groupId>
         <artifactId>powsybl-parent</artifactId>
-        <version>20</version>
+        <version>20.1</version>
```

Can you use the v20.2 instead?
```suggestion
        <version>20.2</version>
```